packages/vuepress-theme-odyssey/package.json

```diff
@@ -5,10 +5,15 @@
   "author": "Okta, Inc.",
   "license": "Apache-2.0",
   "main": "index.js",
+  "scripts": {
+    "lint": "eslint . --ext .js,.vue",
```

Comment:
Can the `--ext` values be moved to our config file?

Reply:
This doesnt appear to be possible, I did a little digging and found this, but its fairly old:

 https://github.com/eslint/eslint/issues/11223

Reply:
Spoke to @jmelberg-okta offline, he referenced this RFC - but work does not been merged for this specific request (as far as I was able to tell) https://github.com/eslint/rfcs/pull/9

Looking at how okta-vue does it, they have taken the same approach as above. https://github.com/okta/okta-vue/blob/master/package.json#L14

In the short term, will not fix